defaults/main.yml

```diff
@@ -13,3 +13,5 @@ datadog_group: root
 
 # default apt repo
 datadog_apt_repo: "deb http://apt.datadoghq.com/ stable main"
+
+datadog_yum_repo: "https://yum.datadoghq.com/rpm/"
```

Comment:
you should use the value of the variable in the template: https://github.com/DataDog/ansible-datadog/blob/1.2.0/templates/datadog.repo.j2#L3-L6